test/serializer/additional-functions/ModifiedBindingsCapturedScopesCalls.js

```diff
@@ -0,0 +1,16 @@
+// add at runtime:var a = 17;
+// Copies of \|\|:2
```

Comment:
I expected this to 1 copy.

Reply:
It's two. The closure is accessed from the optimized function, and the inspect function. But not more!